--- liblqr-0.4.3/configure.ac.orig	2024-08-30 14:12:30.000000000 +0200
+++ liblqr-0.4.3/configure.ac	2025-09-28 20:42:28.289393514 +0200
@@ -58,11 +58,10 @@ AC_CONFIG_HEADERS([config.h])
 
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([-Wall -Werror])
+AM_INIT_AUTOMAKE([1.11 -Wall -Wno-extra-portability foreign])
 
 AC_PROG_CC
 AC_PROG_CXX
-AM_PROG_CC_STDC
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
